client/lib/feed-post-store/post-batch-fetcher.js

```diff
@@ -88,8 +88,8 @@ assign( PostBatchFetcher.prototype, {
 					return;
 				}
 
-				containerId = +match[ 1 ];
-				postId = +match[ 2 ];
+				containerId = match[ 1 ];
```

Comment:
have to be very careful with this one... Maybe check it with `isNumeric` from lodash and convert to a number if true